ftdc/cmd/parser.go

```diff
@@ -69,12 +84,25 @@ type graphOptions struct {
 	// timestamp. The default is all datapoints (minTimeSeconds: 0, maxTimeSeconds: MaxInt64).
 	minTimeSeconds int64
 	maxTimeSeconds int64
+
+	// hideAllZeroes will omit graphs where all values are 0. Generally speaking, graphs that only
+	// contain zeroes won't correlate to anything interesting. A lot of graphs (e.g: component
+	// `State`s) are all zeroes. Avoid graphing them to help users navigate the data that's more
+	// interesting.
+	hideAllZeroes bool
+
+	// vertLinesAtSeconds will draw a vertical line for each element. The items are expected to be
+	// in units of seconds since the epoch.These are to represent "events" that are interest of a
+	// user. Where the user would like to find correlations in other metrics.
```

Comment:
```suggestion
	// vertLinesAtSeconds will draw a vertical line for each element. The items are expected to be
	// in units of seconds since the epoch. These are to represent "events" that are of interest to a
	// user (where the user would like to find correlations in other metrics.)
```